_includes/workshop_table.md

```diff
@@ -29,22 +29,32 @@ Click on an individual event to learn more about that event, including contact i
     </td>
 
     <td>
-        {% unless w.country == "" or w.country == "W3" %}
-      <img src="{{site.url}}/assets/img/flags/{{site.flag_size}}/{{w.country | downcase}}.png" title="{{w.country | replace: '-', ' '}}" alt="{{w.country | replace: '-', ' ' | downcase}}"  class="flags"/>
-      {% endunless %}
 
-      {% if tags contains "online" %}
+      {% if w.online == "Yes" %}
```

Comment:
The proposed change will throw an error if "online" is undefined. The current test, ` {% if tags contains "online" %}`, is an important pre-test.

Reply:
It can't be undefined as it is a column header in the form. In our data there are (currently) no tags, these only come from amy workshops